client/src/components/AnswersAndVoters.tsx

```diff
@@ -0,0 +1,16 @@
+interface Answer {
+  [key: string]: number;
+}
+
+export default function AnswersAndVotes({ answers }: { answers: Answer }) {
+  return (
+    <div className="space-y-2 mb-4">
+      {Object.entries(answers).map(([answer, count]) => (
+        <div key={answer} className="flex justify-between">
+          <span className="text-gray-700">{answer}</span>
+          <span className="font-medium text-gray-900">Votes: {count}</span>
```

Comment:
why you name, `votes` everywhere? there are points

Reply:
I will change that right now